var/spack/repos/builtin/packages/proj/package.py

```diff
@@ -6,7 +6,7 @@
 from spack.package import *
 
 
-class Proj(AutotoolsPackage):
+class Proj(CMakePackage):
```

Comment:
Not all versions support CMake. You'll need to do a multi-build system like with libtiff.

Reply:
Thought I had checked all of them, thanks for catching this!